torchgeo/transforms/transforms.py

```diff
@@ -102,7 +102,13 @@ def forward(self, batch: dict[str, Any]) -> dict[str, Any]:
             batch['boxes'] = Boxes(batch['boxes']).to_tensor(mode='xyxy')
 
         # Torchmetrics does not support masks with a channel dimension
-        if 'mask' in batch and batch['mask'].shape[1] == 1:
+        # Kornia adds a temporal dimension to mask when passed through VideoSequential.
```

Comment:
We should no longer be using our `AugmentationSequential` wrapper, let's use the Kornia version upstream.

Reply:
Got it, I've now switched it to `K.AugmentationSequential`.